src/f32/funcs.rs

```diff
@@ -349,3 +465,25 @@ fn test_scalar_sin_cos() {
     assert!(s.is_nan());
     assert!(c.is_nan());
 }
+
+#[test]
+#[cfg(vec4sse2)]
+fn test_sse2_m128_sin() {
+    use crate::Vec4;
+    use std::f32::consts::PI;
+
+    fn test_sse2_m128_sin_angle(a: f32) {
+        let v = Vec4::splat(a);
+        let v = unsafe { Vec4(sse2::m128_sin(v.0)) };
```

Comment:
Does this function return the sin calculation in each lane? Can you pass in different angles in each lane in the input and get different results in each lane? Might be worth commenting one way or another. If it is actually returning a result in each lane then the test should test all lanes.